src/module/tours/sr5Tours.ts

```diff
@@ -15,22 +15,21 @@ export default class Sr5Tour extends Tour {
     async _preStep() {
         await super._preStep();
 
-        //check if the tour actor already exists. This can happen when resuming the tour
-        // @ts-expect-error
-        this.actor = game.actors?.getName("Tour " + this.id);
-
         //create actor if needed
         if(this.actor == undefined) {
-            this.actor = await Actor.create({
+            this.actor = new SR5Actor.implementation({
                 //@ts-expect-error
                 name: "Tour " + this.id,
                 // @ts-expect-error
-                type: this.config.actorType
-            });
+                type: this.config.actorType,
+                ownership: {
+                    default: 3
+                }
+            }) as SR5Actor;
         }
 
-        //@ts-expect-error Calling _render because it's async unlike render
-        await this.actor.sheet?._render(true);
+        // @ts-expect-error
+        await this.actor.sheet?._render(true, {editable: false});
```

Comment:
If you are curious what we can do to avoid these @ts-expect-error:

https://github.com/League-of-Foundry-Developers/foundry-vtt-types/wiki/A-Quick-Guide-to-Declaration-Merging

I've used this recently for fromUuidSync and added it to global.d.ts

Reply:
I tried it and did not get it to work

Reply:
It can be a bit complicated at first, since the correct namespace and interface within that must be used. So for some of these, one has to seriously dive deep into foundry-vtt-types internals. I've had similar experiences with it.